docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
+. [[ref:27]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 4.1. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:28]] object management group (2014) Ontology definition metamodel . OMG. Available at: https://www.omg.org/spec/ODM/1.1/PDF (Accessed: January 2, 2023).
+. [[ref:29]]  SEMICEU-toolchain-manual. SEMICEU. Available at: https://github.com/SEMICeu/toolchain-manual (Accessed: January 3, 2023).
+. [[ref:30]] Costetchi, E. (2020) eProcurement UML conceptual model conventions. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf (Accessed: January 2, 2023).
+. [[ref:31]] Costetchi, E. (2020) Transformation of the eProcurement UML model into a formal OWL ontology. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml2owl-transformation/uml2owl-transformation.pdf.
+. [[ref:32]] P. Archer, S. Goedertier, and N. Loutas. D7.1.3 - study on persistent uris, with
+identification of best practices and recommendations on the topic for the mss
+and the ec. Deliverable, ISA programme of the European Commission, 2012
+. [[ref:33]] D. Wood, R. Cyganiak, and M. Lanthaler. RDF 1.1 concepts
+and abstract syntax. W3C recommendation, W3C, Feb. 2014.
+http://www.w3.org/TR/2014/REC-rdf11-concepts-20140225/.
+. [[ref:34]] T. Bray, M. Sperberg-McQueen, E. Maler, F. Yergeau, and J. Paoli. Extensible
+markup language (XML) 1.0 (fifth edition). W3C recommendation, W3C, Nov.
+2008. http://www.w3.org/TR/2008/REC-xml-20081126/.
+. [[ref:35]] PwC EU Services. D3.1 - process and methodology for core vocabularies. Deliverable, ISA programme of the European Commission, 2011.
+. [[ref:36]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 3.2. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:37]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.20. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:38]] Bray, T. et al. (eds) (2009) Namespaces in XML 1.0 (Third Edition), Namespaces in XML 1.0 (third edition). Available at: https://www.w3.org/TR/REC-xml-names/.
+. [[ref:39]] P. Saint-Andre and J. Klensin. Uniform resource names (urns). Internet Engineering Task Force (IETF), RFC, 8141, 2017
+. [[ref:40]] Beckett, D. et al. (2014) RDF 1.1 turtle. W3C. Available at: https://www.w3.org/TR/turtle/#sec-iri (Accessed: January 2, 2023).
+. [[ref:41]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.1.7. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:42]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.3. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
```

Comment:
Please use a single reference for OSLO modelling rules, the section shall be included in the citation, and not the reference. @andreea-pasare can you please help find usages and unify the references?

Reply:
in this case, we should update all references  in the text